cypress/e2e/awx/infrastructure/instance-groups/instanceGroups.cy.ts

```diff
@@ -63,7 +63,7 @@ describe(`Instance Groups`, () => {
       cy.get('[data-cy="max-forks"]').clear();
       cy.get('[data-cy="max-forks"]').type('4');
       cy.intercept('POST', awxAPI`/instance_groups/`).as('createInstanceGroup');
-      cy.clickButton(`Create Instance Group`);
+      cy.clickButton(`Create instance group`);
```

Comment:
Looks like this button still has the old text? The test is failing here.

Reply:
thank you! fixing